addons/docs/tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "extends": "../../tsconfig.json",
   "compilerOptions": {
     "rootDir": "./src",
+    "skipLibCheck": true,
```

Comment:
This package is too complex, and contains too many issues, skipping this one for now.. Also I think @shilman is working on a replacement.. should not invest time at this point.